miqa/core/rest/global_settings.py

```diff
@@ -0,0 +1,74 @@
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers, status
+from rest_framework.decorators import action
+from rest_framework.permissions import BasePermission, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from miqa.core.models import GlobalSettings
+from miqa.core.tasks import export_data, import_data
+
+
+class IsSuperUser(BasePermission):
+    def has_permission(self, request, view):
+        return bool(request.user and request.user.is_superuser)
+
+
+class GlobalSettingsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = GlobalSettings
+        exclude = ['id']
+
+
+class GlobalSettingsViewSet(ViewSet):
+    def get_object(self):
+        return GlobalSettings.load()
+
+    def get_permissions(self):
+        if self.request.method == 'GET':
+            return [IsAuthenticated()]
+        else:
+            return [IsSuperUser()]
```

Comment:
Are these supposed to be instances of the permission classes, or the classes themselves?

Reply:
It appears to work either way, but since the DRF docs use the classes themselves, I'll make the change.

Reply:
Changed in [93922ab](https://github.com/OpenImaging/miqa/pull/294/commits/93922abd8d5a2b039cbe50e012cc63e1b8f4ef8a)

Reply:
Actually, scratch that. The permission classes should be uninstantiated when specified as `permission_classes=[cls1, cls2]` like in the docs, but when overriding the `get_permissions` function on a Viewset, the return value should be a list of permission class instances. The change in [93922ab](https://github.com/OpenImaging/miqa/pull/294/commits/93922abd8d5a2b039cbe50e012cc63e1b8f4ef8a) was a breaking change: [See tests](https://github.com/OpenImaging/miqa/runs/5055106927?check_suite_focus=true)